BSP/STM32F413/Src/BSP_CAN.c

```diff
@@ -241,4 +241,4 @@ void CAN1_TX_IRQHandler(void) {
     #ifdef RTOS
     OSIntExit();      // Signal to uC/OS
     #endif
-}
+}
```

Comment:
Don't think you needed to change this file.

Reply:
Yeah was just a typo. There weren't any content modifications made.